pkg/config/config.go

```diff
@@ -149,6 +149,7 @@ func (nv NameValue) GetSecretValue() string {
 }
 
 func readFromKubernetesSecret(secName, keyName string) (string, error) {
+	log.Println("hi")
```

Comment:
is it ??

Reply:
sorry, forgot to remove that, was using it to see if the function was being called